HISTORY.md

```diff
@@ -6,6 +6,12 @@ dev
 
 -   \[Short description of non-trivial change.\]
 
+**Dependencies**
+
+- Switch chardet for the MIT-licensed charset_normalizer for Python3 to remove license
+  ambiguity for projects bundling requests. Python2 still depends upon the
+  LGPL'd module.
+
```

Comment:
Maybe precise that if chardet is installed, it will still be used as it could confuse users.

Reply:
Added.

Reply:
Thanks!